composer.json

```diff
@@ -16,18 +16,18 @@
         "rss": "https://github.com/laminas/laminas-recaptcha/releases.atom"
     },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

Comment:
imo, I think it is time to remove php `~8.0.0` support, other components no longer have php `~8.0.0` :)